internal/pkg/project/project.go

```diff
@@ -47,10 +47,6 @@ func (p *Project) Fs() filesystem.Fs {
 	return p.fs
 }
 
-func (p *Project) FileLoader() filesystem.FileLoader {
-	return p.fileLoader
-}
-
```

Comment:
Povodne som chcel aby `Project/Template` obsahovali factory pre `FileLoader`.
Ale nakoniec som to vyriesil cez `mapper` - je to lepsie, pretoze je tam aj save aj load spolu.
Preto tieto metody odstranujem.

Project/Template uz obsahuje factory `MappersFor`, a tak nam metodu `FileLoader` uz netreba.